index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./css/style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <main>
+    <div class="background-image"><img src="./images/Illustration.png"/ alt="Vector image"></div>
```

Comment:
I think it would be good to include this image as a background image instead of using the `img` tag, It's good to include images of aesthetic value in a design as a background image and since this image is of an aesthetic value kindly include it as a background image.

The `img` tag should be used for images that have a functional purpose in our designs e.g clickable buttons or images used as part of the content in an article.